torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -39,8 +39,10 @@ bool scheduleFusion(Fusion* fusion) {
   // Run through outputs, grab all inputs of outputs
   // squeeze with computeAt to set overall structure.
   for (auto output : fusion->outputs()) {
-    if (output->getValType() != ValType::TensorView)
+    if (output->getValType() != ValType::TensorView ||
+        output->as<TensorView>()->nDims() == 0) {
```

Comment:
I likely need to consider this kind of logic in reduction/normalization schedulers.

Reply:
Yes, AFAIK, I haven't enabled reduce to a scalar tensor yet.
Feel free to leave it as-is and I can patch them when I get to that.